kmath-tensors/src/commonMain/kotlin/space/kscience/kmath/tensors/core/DoubleTensorAlgebra.kt

```diff
@@ -932,6 +921,8 @@ public open class DoubleTensorAlgebra :
     }
 
     override fun Tensor<Double>.lu(): Triple<DoubleTensor, DoubleTensor, DoubleTensor> = lu(1e-9)
+
+    override fun Tensor<Double>.unaryPlus(): Tensor<Double> = copy()
```

Comment:
This one is not obvious. It should be either documented or it should return self. Since tensors are mutable, returning copy is not identity.